diff --git a/engine/solver/det_solver.py b/engine/solver/det_solver.py
index b9abf22..ed83bdf 100644
--- a/engine/solver/det_solver.py
+++ b/engine/solver/det_solver.py
@@ -96,7 +96,7 @@ class DetSolver(BaseSolver):
 
             self.last_epoch += 1
 
-            if self.output_dir and epoch < self.train_dataloader.collate_fn.stop_epoch:
+            if self.output_dir:
                 checkpoint_paths = [self.output_dir / 'last.pth']
                 # extra checkpoint before LR drop and every 100 epochs
                 if (epoch + 1) % args.checkpoint_freq == 0:
